.github/workflows/test-alternative-compilers.yml

```diff
@@ -3,7 +3,7 @@ name: run gsw_check (alternative compilers)
 on:
   pull_request:
   push:
-    #branches: [master]
+    branches: [master]
```

Comment:
```suggestion
    branches: [main]
```

Reply:
adapted the new branch name in all workflows